lib/matplotlib/dviread.py

```diff
@@ -1015,11 +1017,16 @@ def find_tex_file(filename, format=None):
     if format is not None:
         cmd += ['--format=' + format]
     cmd += [filename]
-    _log.debug('find_tex_file(%s): %s', filename, cmd)
-    pipe = subprocess.Popen(cmd, stdout=subprocess.PIPE)
-    result = pipe.communicate()[0].rstrip()
-    _log.debug('find_tex_file result: %s', result)
-    return result.decode('ascii')
+    try:  # Below: strip final newline.
+        result = cbook._check_and_log_subprocess(cmd, _log)[:-1]
+    except RuntimeError:
```

Comment:
I think the exception needs logging.

Reply:
https://travis-ci.org/anntzer/matplotlib/jobs/432616612#L4359 (for example) is what happens if we make find_tex_file throw an exception on missing files.  Now there may be other hidden bugs due to that, but at least this PR is not changing the semantics.  However I think in most cases above we'll still get an exception, just later...

cbook._check_and_log_subprocess already performs logging.